src/quo2/components/wallet/network_routing/animation.cljs

```diff
@@ -0,0 +1,114 @@
+(ns quo2.components.wallet.network-routing.animation
+  (:require [react-native.reanimated :as reanimated]))
+
+(def ^:private slider-timing 300)
+
+(defn show-slider
+  [opacity-shared-value]
+  (reanimated/animate opacity-shared-value 1 slider-timing))
+
+(defn hide-slider
+  [opacity-shared-value]
+  (reanimated/animate opacity-shared-value 0 slider-timing))
+
+(defn increase-slider
+  [width-shared-value height-shared-value]
+  (reanimated/animate width-shared-value 8 slider-timing)
+  (reanimated/animate height-shared-value 40 slider-timing))
+
+(defn decrease-slider
+  [width-shared-value height-shared-value]
+  (reanimated/animate width-shared-value 4 slider-timing)
+  (reanimated/animate height-shared-value 32 slider-timing))
+
+(def ^:private pressed-bar-timing 600)
+
+(defn move-previous-bars
+  [{:keys [bars bars-widths-negative]}]
+  (doseq [[bar-idx bar] (map-indexed vector bars)
+          :let          [new-translation-x (->> (take (inc bar-idx) bars-widths-negative)
+                                                (reduce +)
+                                                (dec))]]
+    (reanimated/animate (:translate-x-shared-value bar)
+                        new-translation-x
+                        pressed-bar-timing)))
+
+(defn move-pressed-bar
+  [{:keys                              [bars-widths-negative number-previous-bars]
+    {:keys [translate-x-shared-value]} :bar}]
+  (let [new-translation-x (reduce + (take number-previous-bars bars-widths-negative))]
+    (reanimated/animate translate-x-shared-value
+                        new-translation-x
+                        (- pressed-bar-timing 20))))
+
+(defn move-next-bars
+  [{:keys [bars bars-widths-negative number-previous-bars extra-offset add-new-timeout]}]
+  (doseq [[bar-idx bar] (map-indexed vector bars)
+          :let          [number-bars-before (+ number-previous-bars
+                                               (inc bar-idx))
+                         new-translation-x  (->> (take number-bars-before bars-widths-negative)
+                                                 (reduce +)
+                                                 (* 1.05))]]
+    (reanimated/animate (:translate-x-shared-value bar) new-translation-x pressed-bar-timing)
+    (add-new-timeout
+     (keyword (str "fix-next-bars-position-" bar-idx))
+     #(let [translate-x-value (reanimated/get-shared-value (:translate-x-shared-value bar))
```

Comment:
@ulisesmac `#()` syntax is to preferred only for very simple functions, usually one-liners. I can't find sources to my claim, but most good Clojure code in the wild more or less stick to this idea. In this PR I see `#()` being used a few times in quite extensive functions.

When we use `#()` we impact readability, especially in **longer functions**, we are:

- Forcing the developer to scan the entire function body to see if any argument is being used. This is painful, `(fn [] ...)` would have clearly communicated the anonymous function doesn't rely on any argument from the callback. 
- Hiding the function signature, making the code less clear.
- Making the multiline function less ergonomic to change if the dev needs to respect the Clojure Style Guide that recommends against using `#()` with more than one form https://guide.clojure.style/#no-multiple-forms-fn-literals.

I believe our guidelines should more clearly specify this suggestion.

Reply:
:thinking: 

I agree with it. I will change this one and the following one you suggested. I wouldn't set it as a rule, but it's true, for long functions it's better to use  `*(fn [])` :+1: